cinema_service/settings.py

```diff
@@ -20,12 +21,10 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
```

Comment:
Security Issue: It's not a good practice to store the SECRET_KEY in your settings file, especially if your code is version controlled and publicly accessible. Consider using environment variables to store your secret key. You can use python-decouple or django-environ to achieve this.